frontend/projects/upgrade/src/app/features/dashboard/feature-flags/pages/feature-flag-root-page/feature-flag-root-page-content/feature-flag-root-section-card/feature-flag-root-section-card-table/feature-flag-root-section-card-table.component.html

```diff
@@ -1,6 +1,6 @@
 <div class="flags-list-container">
-  <div scroll (scrolled)="fetchFlagsOnScroll()" class="flags-list-table-container" #tableContainer>
-    <mat-progress-bar class="spinner" mode="indeterminate" *ngIf="isLoading$ | async"></mat-progress-bar>
+   <div (wheel)="fetchFlagsOnScroll()" class="flags-list-table-container" #tableContainer>
+    <mat-spinner class="spinner" mode="indeterminate" *ngIf="isLoading$ | async"></mat-spinner>
```

Comment:
@zackcl @Yagnik56 now that the mat-progress-bar is showing up correctly in our table elements (the progress bar at top of the table border that animates while loading), do we feel like this should remain `mat-progress-bar`?

Reply:
@KD1712 per the discussion in mtg yesterday, we want to keep it as mat-progress-bar, i think this PR is ready otherwise